src/Symfony/Component/Validator/Constraints/YamlValidator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml as SymfonyYaml;
+
+/**
+ * @author Mokhtar Tlili <tlili.mokhtar@gmail.com>
+ */
+class YamlValidator extends AbstractFormatValidator
+{
+    protected function validateFormat(string $value): bool
+    {
+        try {
+            SymfonyYaml::parse($value);
```

Comment:
Consider adding some flags to make it recognize custom tags like `!php/const` and so on